### PR TITLE
Allow to use pluralized translations with block stats widgets

### PR DESCRIPTION
**Pull request by phansys**
## Subject

Allow to use pluralized translations with block stats widgets.

I am targeting this branch, because this change respects BC.

## Changelog

```markdown
### Added
- Allow to use pluralized translations with text provided through `sonata_admin.dashboard.blocks.{offset}.settings.text`.
```
## To do
   
- [x] ~~Add tests~~ (I don't think it could be possible without testing [responsibilities which aren't part of this bundle](https://github.com/symfony/twig-bridge/blob/a265f8ddc0d9a2787f7683270a859c83893e4a94/Tests/Extension/TranslationExtensionTest.php#L35-L48));
- [x] Update the documentation.

### Example:
```xml
<!-- translations/messages.xlf -->

<trans-unit id="admin.block.stats.results">
    <source>admin.block.stats.results</source>
    <target>{0} Results|{1} Result|]1,Inf[ Results</target>
</trans-unit>
```

image

### CODE REVIEW

File: docs/reference/dashboard.rst [hunk 1]
```diff
@@ -319,11 +319,34 @@ counter is related to the filters from one admin
                         settings:
                             code:  sonata.page.admin.page    # admin code - service id
                             icon:  fa-magic                  # font awesome icon
-                            text:  Edited Pages
+                            text:  app.page.stats            # static text or translation message
                             color: bg-yellow                 # colors: bg-green, bg-red and bg-aqua
                             filters:                         # filter values
                                 edited: { value: 1 }
 
+The block configuration for "settings.text" accepts static text or a translation message,
```
Comment (greg0ire) [inline docs/reference/dashboard.rst]:
```suggestion
The block configuration for `settings.text` accepts static text or a translation message,
```

File: docs/reference/dashboard.rst [hunk 2]
```diff
@@ -319,11 +319,34 @@ counter is related to the filters from one admin
...
+.. code-block:: xml
+
+    <!-- messages.en.xlf -->
+
+    <trans-unit id="app.page.stats">
+        <source>app.page.stats</source>
+        <target>{0} results|{1} result|]1,Inf] results</target>
+    </trans-unit>
+
+**If you're using "symfony/translation" >= 4.2, you can also opt for the ICU Message Format**
```
Comment (greg0ire) [inline docs/reference/dashboard.rst]:
```suggestion
**If you're using `symfony/translation` >= 4.2, you can also opt for the ICU Message Format**
```

Comment (phansys) [reply docs/reference/dashboard.rst]: Comments addressed :+1:

File: docs/reference/dashboard.rst [hunk 3]
```diff
@@ -319,11 +319,34 @@ counter is related to the filters from one admin
...
+.. code-block:: xml
+
+    <!-- messages.en.xlf -->
+
+    <trans-unit id="app.page.stats">
+        <source>app.page.stats</source>
+        <target>{0} results|{1} result|]1,Inf] results</target>
+    </trans-unit>
+
+**If you're using ``symfony/translation`` >= 4.2, you can also opt for the ICU Message Format**
```
Comment (OskarStark) [inline docs/reference/dashboard.rst]: opt ... opt in?

**Comment by greg0ire** [comment]
Thanks @phansys !

**Pull request merged**